lib/utils/parse-incomplete-markdown.ts

```
@@ -0,0 +1,989 @@
+const linkImagePattern = /(!?\[)([^\]]*?)$/;
+const boldPattern = /(\*\*)([^]*?)$/;
```

Biome (2.1.2)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)